frontend/wasm/src/editor_api.rs

```diff
@@ -152,6 +152,11 @@ impl EditorHandle {
 						for message in editor.handle_message(AnimationMessage::IncrementFrameCounter) {
 							handle.send_frontend_message_to_js(message);
 						}
+
+						// Currently needed for auto panning to work. Maybe there's a better way to do this
+						for message in editor.handle_message(BroadcastMessage::TriggerEvent(BroadcastEvent::AnimationFrame)) {
+							handle.send_frontend_message_to_js(message);
+						}
```

Comment:
I think it would be better to dispatch both the broadcast event and the increment frame counter message from the CurrentTime input preprocessor message handler. Having these multiple invocations is not pretty. It also somewhat obfuscates that setting the current time now has a side effect (of changing the frame time).
I'm also wondering if it even makes sense to use the animation frame here at all for non render related editor timing. We could consider adding a LogicTick  broadcast event which is triggered on a set interval timer and seperate this completely from animation related functionality

Reply:
> I'm also wondering if it even makes sense to use the animation frame here at all for non render related editor timing. We could consider adding a LogicTick broadcast event which is triggered on a set interval timer and seperate this completely from animation related functionality

(To be clear, that would be a separate discussion for a later PR, this one is just focused on un-breaking the functionality without refactoring or rearchitecting it.)

Reply:
I mean the code should work, I just don't like it, but I didn't like it before either so that is not a merge blocker